themes/nu4a.omp.json

```diff
@@ -88,6 +88,11 @@
       "type": "rprompt"
     }
   ],
+  "secondary_prompt": {
+    "background": "#ec9706",
+    "foreground": "#100e23",
+    "template": "  "
```

Comment:
why an empty block?

Reply:
Wait, it's not empty, but a direct copy of the character (thus doesn't show on GH). @sectorogo can you add the `\uxxxx` reference?

Reply:
Yes, i replaced the character with his unicode number \uf061